### PR TITLE
Enhancement/implement usecontext

### LINKED ISSUE #47: Implement useContext
Implement useContext for user query in dashboard.js, dashboardlist.js and userprofile.js

### PR DESCRIPTION
**Pull request by tfoxcollis**
## Summary:
Implement useContext for user query in dashboard.js, dashboardlist.js, and userprofile.js

## Type of Changes:
- [ ] Bug fix
- [x] Refactor
- [x] New feature

## How was this tested:
chrome dev tools

## What are the relevant tickets:
closes #47

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -34,7 +39,7 @@ const App = () =>  {
           <UserProfile />
         </Route>
       </div>
-    </ApolloProvider>
+    </UserContext.Provider>
```
Comment (BlueNealis) [inline src/Components/App/App.js:42]: Nice!How you set this up makes sense its easy to understand.

File: src/Context/UserContext.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from "react"
+
+const UserContext = React.createContext()
+
```
Comment (BlueNealis) [inline src/Context/UserContext.js:4]: So then does the data you pass in from the app query get added here? Nice!

**Review by BlueNealis — APPROVED** [review]
You are a superstar

File: src/Context/UserContext.js (hunk 2 above)
Comment (tfoxcollis) [reply src/Context/UserContext.js:4]: Yes!  The data is being added in app.js.  Havent figured out how to change or update context yet.  Going to look into adding more of the methods that we are passing around a lot to useContext.

**Pull request merged**